plugin/plugin.go

```diff
@@ -162,6 +137,108 @@ func (e *Ratelimit) Type() rules.ActionType {
 	return rules.ActionTypeNondisruptive
 }
 
+func (e *Ratelimit) parseConfig(config string) error {
+	// acceptable keys
+	var err error
+
+	tokens := strings.Split(config, "&")
+
+	requiredValues := map[string]bool{
+		"zone":   false,
+		"events": false,
+		"window": false,
+	}
+
+	for _, token := range tokens {
+		pair := strings.Split(token, "=")
```

Comment:
Probably better to use `strings.Cut`

Reply:
done and and added a test case for it as well